index.html

```diff
@@ -14,6 +14,7 @@
   <script src="assets/ext/angular-cookies.min.js"></script>
   <script src="assets/ext/jquery.min.js"></script>
   <script src="assets/ext/bootstrap.min.js"></script>
+  <script src="assets/js/config.example.js"></script>

```

Comment:
Could that be a way we can achieve for that? http://stackoverflow.com/a/24328341/763907